zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -313,7 +312,7 @@ where
 
             // The tip estimate may not be the same as the one coming from the state
             // but this is ok for an estimate
-            let (estimated_distance_to_chain_tip, tip_height) = latest_chain_tip
+            let (estimated_distance_to_chain_tip, estimated_tip_height) = latest_chain_tip
```

Comment:
I'm not sure about that, it is an estimate based on the local clock, rather than the height of any actual block we've seen.

It can be very different from the network tip if the local clock is wrong.

Reply:
That may have been bad naming on my part, it's the actual best chain tip height from `best_tip_height_and_block_time`.

Reply:
I made the changes I had in mind in https://github.com/ZcashFoundation/zebra/pull/5809 as a suggestion PR, feel free to close it if it might cause conflicts with the refactor.

Reply:
Ah right, so this probably needs to be called `local_tip_height`, to distinguish it from the estimated distance to the network tip.